### ISSUE TITLE
Wrong Implementation

### ISSUE THREAD

**Issue by sungyoon-lee** [meliketoy/Pytorch-wide-resnet#9]
@meliketoy

1. BatchNorm2d
```
self.bn1 = nn.BatchNorm2d(nStages[3], momentum=0.9)
```
In the official Pytorch code, they use the default value of momentum, momentum=0.1.
https://github.com/szagoruyko/wide-residual-networks/blob/ae6d0d0561484172790c7a63c8ce6ade5a5a2914/pytorch/utils.py#L55
I think you've confused with the momentum used in Tensorflow which has different meanings.
https://stackoverflow.com/questions/48345857/batchnorm-momentum-convention-pytorch

2. AvgPool
```
out = F.avg_pool2d(out, 8)
```
In the official Pytorch code, they use F.avg_pool2d(out, 8, 1, 0), but you've used the default stride which is same as the kernel size = 8.
https://github.com/szagoruyko/wide-residual-networks/blob/master/pytorch/resnet.py#L56

**Comment by sungyoon-lee** [comment]
I found that it doesn't matter in the case of AvgPool since the input has a dimension of (batch_size, 64, 1, 1).

**Comment by y0ast** [comment]
Also the `conv`s are not supposed to have a bias. At the moment this cannot be trusted to do the same as the official WRN.
